media/js/firefox/new.js

```diff
@@ -171,12 +171,39 @@
                     $thankYou.focus();
                 }, 500);
             }
+
+            // Added measurements fire only when the download is fired
+            spring_campaign_measurement();
         }
 
         function show_scene_anim(scene) {
             show_scene(scene, true);
         }
 
+        // Pixel to be removed on July 31st, 2015 (Bug 1168440)
+        function spring_campaign_measurement () {
+            var _dntStatus = navigator.doNotTrack || navigator.msDoNotTrack;
+            var fxMatch = navigator.userAgent.match(/Firefox\/(\d+)/);
+
+            if (fxMatch && Number(fxMatch[1]) < 32) {
+                // Can't say for sure if it is 1 or 0, due to Fx bug 887703
+                _dntStatus = (_dntStatus === 'yes') ? 'Unknown' : 'Unspecified';
+            } else {
+                _dntStatus = { '0': 'Disabled', '1': 'Enabled' }[_dntStatus] || 'Unspecified';
+            }
+
+            if (_dntStatus !== 'Enabled'){
```

Comment:
@schalkneethling thanks for this. Guidance from the privacy team is that DNT should be tied to user intention, rather than default browser settings.